motionplan/planManager.go

```diff
@@ -92,6 +92,23 @@ func (pm *planManager) PlanSingleWaypoint(ctx context.Context,
 
 	var cancel func()
 
+	if pathdebug {
```

Comment:
Same as the other comment; I don't think `String()` is appropriate here, since this is specific output for an internal debug tool and not desirable outside of that context.